clients/tests/test_spmm_bell.cpp

```diff
@@ -25,6 +25,7 @@
 
 #include <hipsparse.h>
 
+#if(!defined(CUDART_VERSION))
```

Comment:
spmm for bell format is only supported with CUDA_R_16F for matrices A and B when using compute capability 7.0. On the newest GPU's CUDA_R_32F becomes supported. Therefore turning of cuda tests.